packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.Props.ts

```diff
@@ -31,34 +36,14 @@ export interface IActivityItemProps extends React.AllHTMLAttributes<HTMLElement>
   isCompact?: boolean;
 
   /**
-   * A renderer for the description of the current activity.
-   */
-  onRenderActivityDescription?: IRenderFunction<IActivityItemProps>;
-
-  /**
-   * A renderer that adds the text of a comment below the activity description.
-   */
-  onRenderComments?: IRenderFunction<IActivityItemProps>;
-
-  /**
-   * A renderer to create the icon next to the activity item.
-   */
-  onRenderIcon?: IRenderFunction<IActivityItemProps>;
-
-  /**
-   * A renderer adds a time stamp. If not included, timeStamp is shown as plain text below the activity.
-   */
-  onRenderTimeStamp?: IRenderFunction<IActivityItemProps>;
```

Comment:
I would keep these all in place. They're fine to have if the consumer needs to completely alter visual presentation. The goal of having assignable data fields that take JSX is to make it faster to internally-format the data without creating new `function` objects on every `render` pass. If you just add the new fields, you only need a new minor version.